08_Support_Vector_Machines/00_Exercises Readme.ipynb

```diff
@@ -229,6 +236,60 @@
     "Note: Here you'll have to use the math behind the SVM!"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "$$ w = \\begin{bmatrix}0\\\\1\\end{bmatrix}$$\n",
+    "\n",
+    "$$ b = -1$$\n",
+    "\n",
+    "1. $$ \\text{minimize}\\ \\frac{1}{2}\\ \\begin{bmatrix}0&1\\end{bmatrix} \\begin{bmatrix}0\\\\1\\end{bmatrix}$$\n",
+    "\n",
+    "Note:\n",
+    "\n",
+    "$$\\vec{x}_1 = \\begin{bmatrix}0\\\\2\\end{bmatrix} $$\n",
+    "$$\\vec{x}_2 = \\begin{bmatrix}0\\\\0\\end{bmatrix} $$\n",
+    "$$\\vec{x}_3 = \\begin{bmatrix}3\\\\0\\end{bmatrix} $$\n",
+    "$$\\vec{x}_4 = \\begin{bmatrix}2\\\\3\\end{bmatrix} $$\n",
+    "\n",
+    "Lagrangian multiplier\n",
+    "\n",
+    "2. $$ L(\\vec{w},b,\\alpha_1,...,\\alpha_n) = \\frac{1}{2}\\vec{w}^T \\vec{w} - \\sum_{i=1}^n \\alpha_i [(\\vec{w}^T \\vec{x}_i + b)y_i - 1]$$\n",
+    "\n",
+    "$$ L(\\vec{w},b,\\alpha_1,\\alpha_2,\\alpha_3,\\alpha_4)\\\\\n",
+    "= \\frac{1}{2}\\begin{bmatrix}0&1\\end{bmatrix} \\begin{bmatrix}0\\\\1\\end{bmatrix} \n",
+    "- \\left(\n",
+    "\\alpha_1\\cdot(\\begin{bmatrix}0&1\\end{bmatrix}\\cdot \\begin{bmatrix}0\\\\2\\end{bmatrix} + 1) \\cdot 1 - 1 + \n",
+    "\\alpha_2\\cdot(\\begin{bmatrix}0&1\\end{bmatrix}\\cdot \\begin{bmatrix}0\\\\0\\end{bmatrix} + 1) \\cdot -1 - 1 + \n",
```

Comment:
Add parenthesis around the `* (-1) `  in 8.3.2 (Lagrangian multiplier) for improved readability.